source/types.ts

```diff
@@ -137,6 +137,13 @@ export interface Options<T> {
 	*/
 	clearInvalidConfig?: boolean;
 
+	/**
+	The mode that will be used for the config file.
```

Comment:
Linkify `mode` to a website that explains what `mode` is. Not every user is familiar with the concept.